invenio_jobs/models.py

```diff
@@ -157,8 +157,16 @@ def generate_args(cls, job, task_arguments=None):
         args = Task.get(job.task)._build_task_arguments(
             job_obj=job, **task_arguments or {}
         )
-        args = json.dumps(args, indent=4, sort_keys=True, default=str)
+
+        def custom_json(obj):
```

Comment:
LGTM! There are another [couple of places](https://github.com/inveniosoftware/invenio-jobs/blob/master/invenio_jobs/services/results.py#L53) with `default=str`.
Was that working fine?